tensorzero-core/src/inference/types/extra_body.rs

```diff
@@ -10,7 +11,40 @@ pub struct ExtraBodyConfig {
 #[derive(Clone, Debug, Deserialize, PartialEq, Serialize)]
 pub struct ExtraBodyReplacement {
     pub pointer: String,
-    pub value: Value,
+    #[serde(flatten)]
+    pub kind: ExtraBodyReplacementKind,
+}
+
+#[derive(Clone, Debug, Deserialize, PartialEq, Serialize)]
+#[serde(rename_all = "snake_case")]
+pub enum ExtraBodyReplacementKind {
+    Value(Value),
+    // We only allow `"delete": true` to be set - deserializing `"delete": false` will error
+    #[serde(
+        serialize_with = "serialize_delete",
+        deserialize_with = "deserialize_delete"
+    )]
+    Delete,
```

Comment:
I believe the issue asks for `Null` rather than `Delete`

Reply:
I think that's a different issue (setting explicit `null` through the toml)